### PR TITLE
Storybook bulk channel creator

### PR DESCRIPTION
**Pull request by buoyad**
- Factor out `ModalTitle` that was in the new add to channel modal into `teams/common`
- Add `prefix` prop to `Kb.NewInput` to show an unremovable prefix to the left of the input
- Add bulk channel creator story
image
image

cc @keybase/y2ksquad

### CODE REVIEW

File: shared/common-adapters/plain-input.d.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ export type TextInfo = {
 export type InternalProps = {} & DefaultProps & Props
 
 declare class PlainInput extends React.Component<Props> {
-  defaultProps: DefaultProps
+  static defaultProps: DefaultProps
```
Comment (jakob223) [inline shared/common-adapters/plain-input.d.ts]: what's this do?

File: shared/teams/new-team/wizard/create-channels.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        fullWidth={true}
+        style={styles.body}
+        gap={Styles.isMobile ? 'xsmall' : 'tiny'}
+      >
+        <Kb.Text type="BodySmall">Channels can be joined by anyone in the team, unlike subteams.</Kb.Text>
+        <ChannelInput isGeneral={true} />
+        {channels.map((value, idx) => (
+          <ChannelInput key={idx} onChange={setChannel(idx)} value={value} onClear={() => onClear(idx)} />
+        ))}
+        <Kb.Button mode="Secondary" icon="iconfont-new" tooltip="" onClick={onAdd} />
```
Comment (jakob223) [inline shared/teams/new-team/wizard/create-channels.tsx]: This is a little bit wider in the design. Also, any reason not to include a tooltip?

File: shared/common-adapters/plain-input.d.ts (hunk 1 above)
Comment (buoyad) [reply shared/common-adapters/plain-input.d.ts]: It's more correct, and makes this pass type checking
https://github.com/keybase/client/blob/9781138fe2bd2abce7d980a9727872c34f7183e7/shared/common-adapters/new-input.tsx#L77

File: shared/teams/new-team/wizard/create-channels.tsx (hunk 2 above)
Comment (buoyad) [reply shared/teams/new-team/wizard/create-channels.tsx]: I made it wider. I'm going to lobby to remove the tooltip requirement on Icon buttons

**Comment by buoyad** [comment]
ok @jakob223 addressed feedback, PTAL

**Pull request merged**